.stylelintrc

```diff
@@ -1,6 +1,6 @@
 {
-  "extends": "stylelint-config-standard-scss",
+  "extends": "stylelint-config-tw",
```

Comment:
I didn't find a package called `stylelint-config-tw`.
I found [stylelint-config-tailwindcss](https://github.com/zhilidali/stylelint-config-tailwindcss), but it seems to have not much rules in it.
I advise that just use `stylelint-config-standard` here if you don't want to use sass.

Reply:
updated to  "extends": "stylelint-config-standard-scss",